assets/prometheus-adapter/api-service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1beta1.metrics.k8s.io
+spec:
+  group: metrics.k8s.io
+  groupPriorityMinimum: 100
+  insecureSkipTLSVerify: true
```

Comment:
this can get the serving cert injected as well, just add this annotation:

```
metadata:
  annotations:
    service.alpha.openshift.io/inject-cabundle: "true"
```